src/database.c

```diff
@@ -56,6 +56,7 @@
 static MDB_env *env;
 static MDB_dbi dbi;
 static int dbi_init = 0;
+static unsigned MDB_MAXKEYSIZE;
```

Comment:
I'd rename this to `MDB_maxkeysize` just to be safe. Any future releases of this library which does choose to actually expose MDB_MAXKEYSIZE might result in naming a clash. Also, we don't want to break the `#define` directive name conventions we've employed throughout the project. i.e. when we see the use of all upper case variable names, we automatically know that it refers to some preprocessor directive.